### ISSUE TITLE
How can I do  when Paraview cann't open hdf5?

### ISSUE THREAD

**Issue by Polly0413** [badlands-model/badlands#41]
When I opened the hdf5 results in h5 folder with Paraview, it gave me these messages below. How can I do next?  Thank you very much!
ERROR: In vtkVisItChomboReader.cxx, line 135
vtkVisItChomboReader (000002323C4BCA10): The selected database type was not correct for the given file.  The message from the database was Cannot be a Chombo file, does not have Chombo_global

ERROR: In vtkExecutive.cxx, line 730
vtkPVCompositeDataPipeline (0000023234DFFA30): Algorithm vtkFileSeriesReader (0000023247C2F7D0) returned failure for request: vtkInformation (0000023247917650)
  Debug: Off
  Modified Time: 445492
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FORWARD_DIRECTION: 0
  ALGORITHM_AFTER_FORWARD: 1

ERROR: In vtkAvtSTMDFileFormatAlgorithm.cxx, line 127
vtkVisItChomboReader (000002323C4BCA10): Unable to find any meshes

ERROR: In vtkExecutive.cxx, line 730
vtkPVCompositeDataPipeline (0000023234DFFA30): Algorithm vtkFileSeriesReader (0000023247C2F7D0) returned failure for request: vtkInformation (0000023247917650)
  Debug: Off
  Modified Time: 445492
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FORWARD_DIRECTION: 0
  ALGORITHM_AFTER_FORWARD: 1

ERROR: In vtkAvtSTMDFileFormatAlgorithm.cxx, line 127
vtkVisItChomboReader (000002323C4BCA10): Unable to find any meshes

ERROR: In vtkExecutive.cxx, line 730
vtkPVCompositeDataPipeline (0000023234DFFA30): Algorithm vtkFileSeriesReader (0000023247C2F7D0) returned failure for request: vtkInformation (0000023247917650)
  Debug: Off
  Modified Time: 445492
  Reference Count: 1
  Registered Events: (none)
  Request: REQUEST_DATA_OBJECT
  FORWARD_DIRECTION: 0
  ALGORITHM_AFTER_FORWARD: 1

**Comment by GeoMattB** [comment]
TLDR open the xdmf/xmf files in paraview.

Hdf5 is just a bucket. The metadata that describes where everything is within it in a way that Paraview can use is in the .xmf files.
